content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst

```diff
@@ -4,97 +4,229 @@ Inventory adjustments
+listed, with each product line containing the following information:
+
+.. image:: count_products/count-products-inventory-adjustments-page.png
+   :align: center
+   :alt: Inventory adjustments page.
+
+- :guilabel:`Location`: the specific location in the warehouse where a product is stored.
+- :guilabel:`Product`: the product whose quantity is listed on the inventory adjustment line.
+- :guilabel:`Lot/Serial Number`: the specific tracking identifier assigned to the specific product
+  listed. It can contain letters, numbers, or a combination of both.
+
+  .. note::
+     If a specific product has a quantity of more than `1.00` in stock, and more than one serial
+     number (or lot number) assigned to it, each uniquely-identified product is displayed on its own
+     product line with its own lot/serial number, displayed under the :guilabel:`Lot/Serial Number`
+     column.
+
+- :guilabel:`On Hand Quantity`: the quantity of the product currently recorded in the database.
+- :guilabel:`UoM`: the *unit of measure* in which the product is measured. Unless otherwise
+  specified (i.e., in :guilabel:`Pounds` or :guilabel:`Ounces`), the default :abbr:`UoM (unit of
+  measure)` is :guilabel:`Units`.
+- :guilabel:`Counted Quantity`: the real quantity counted during an inventory count. This field is
+  left blank by default, but can be changed, depending on if it matches the :guilabel:`On Hand
+  Quantity` or not.
+- :guilabel:`Difference`: the difference between the :guilabel:`On Hand Quantity` and
+  :guilabel:`Counted Quantity`, once an inventory adjustment is made.
+- :guilabel:`Scheduled Date`: the date at which a count should be made. If not otherwise specified,
+  this date will default to the 31st of December of the current year.
+- :guilabel:`User`: the person assigned to the count in the database. This can either be the person
+  physically counting the inventory, or applying the count in the database.
+- :guilabel:`Product Category`: the category assigned internally to a specific product. Unless
+  otherwise specified (i.e., as :guilabel:`Consumable` or :guilabel:`Rental`), the default *Product
+  Category* is set to :guilabel:`All`.
+- :guilabel:`Available Quantity`: the quantity of a specific product that is currently available,
+  based on any outstanding/uncompleted sales orders, purchase orders, or manufacturing orders that
+  might change the available quantity once fulfilled.
+- :guilabel:`Accounting Date`: the date on which the adjustments will be accounted in the Odoo
+  *Accounting* app.
+- :guilabel:`Company`: the company whose database these inventory adjustments are being made on. The
+  company is listed in the top right corner of the database, next to the user currently logged in.
+
+.. tip::
+   Some columns are hidden by default. To reveal these columns, click the :guilabel:`additional
+   options` button (three dots icon, at the top right of the header row), and select any desired
```

Comment:
button description formatting - I think this is a stylistic choice to describe the button rather than use the icon, and I think it works well but I feel like the parenthesis are only needed around the icon description. I am curious on @StraubCreative 's thoughts here 👇
```suggestion
   Some columns are hidden by default. To reveal these columns, click the :guilabel:`additional
   options` button (three dots icon) at the top right of the header row, and select any desired
```